# HG changeset patch
# User dnsimon
# Date 1587251123 -7200
#      Sun Apr 19 01:05:23 2020 +0200
# Node ID 9f496d24bdc646528b6dd053cd3c8503e09300cc
# Parent  ee4d5e999653767a63cc85fda0972a86103f1d73
set default VM vendor to GraalVM Community

diff -r ee4d5e999653 -r 9f496d24bdc6 src/share/vm/runtime/arguments.cpp
--- a/src/share/vm/runtime/arguments.cpp	Tue Apr 14 19:55:35 2020 +0100
+++ b/src/share/vm/runtime/arguments.cpp	Sun Apr 19 01:05:23 2020 +0200
@@ -69,7 +69,7 @@
 #ifdef VENDOR_URL_VM_BUG
 # define DEFAULT_VENDOR_URL_BUG VENDOR_URL_VM_BUG
 #else
-# define DEFAULT_VENDOR_URL_BUG "http://bugreport.java.com/bugreport/crash.jsp"
+# define DEFAULT_VENDOR_URL_BUG "https://github.com/oracle/graal/issues"
 #endif
 #define DEFAULT_JAVA_LAUNCHER  "generic"
 
diff -r ee4d5e999653 -r 9f496d24bdc6 src/share/vm/runtime/vm_version.cpp
--- a/src/share/vm/runtime/vm_version.cpp	Tue Apr 14 19:55:35 2020 +0100
+++ b/src/share/vm/runtime/vm_version.cpp	Sun Apr 19 01:05:23 2020 +0200
@@ -142,8 +142,7 @@
 #ifdef VENDOR
   return VENDOR;
 #else
-  return JDK_Version::is_gte_jdk17x_version() ?
-    "Oracle Corporation" : "Sun Microsystems Inc.";
+  return "GraalVM Community";
 #endif
 }
 
